### ISSUE TITLE
Random crash with no error message

### ISSUE THREAD

**Issue by otykier** [otykier/TabularEditor#184]
Some users running Windows 10 have reported that Tabular Editor sometimes crashes out of nowhere, without any visible error messages. If you experience this, please open your Event Viewer under "Windows Logs" > "Application" and see if you can locate any error messages there. One user reported the following error from Event Viewer:

```
Faulting application name: TabularEditor.exe, version: 2.7.6814.17990, time stamp: 0x5b857f7e
Faulting module name: ucrtbase.dll, version: 10.0.17134.191, time stamp: 0xcb91c047
Exception code: 0xc0000409
Fault offset: 0x000a253b
Faulting process id: 0x54c8
Faulting application start time: 0x01d4414f2f5b5b22
Faulting application path: C:\Program Files (x86)\Tabular Editor\TabularEditor.exe
Faulting module path: C:\WINDOWS\System32\ucrtbase.dll
Report Id: 20350983-da0b-4852-a4f7-3cb4294dae42
Faulting package full name: 
Faulting package-relative application ID:
```

If you have a similar error message, try running Tabular Editor in Windows 8 compatibility mode, while setting the DPI scaling setting to "System" (see screenshot below). Please also report back here, if these settings solve the problem or not in your case. Thanks!

![image](https://user-images.githubusercontent.com/8976200/44934669-8a903500-ad22-11e8-8a78-bc71e5e8d251.png)

**Comment by otykier** [comment]
@kalmenk have you tried the provided workaround, and did it solve anything in your case?

**Comment by kalmenk** [comment]
@otykier I haven't tried it but also haven't experienced it crashing in a while.

**Comment by Jeff-Lewis** [comment]
FYI - TabularEditor.exe was crashing immediately on startup for me as well, although might be a different cause. I fixed it by making a backup and removing my `%LocalAppData%\TabularEditor` folder. My preferences.json file was corrupted.

**Comment by albertospelta** [comment]
@otykier I've experienced the same problem with identical error messages found in the Event Viewer.

The error is very annoying and very frequent if you copy text from external applications and paste it into the expression editor. It seems like this issue is related to the COM threading model.

The thread which  reads from the `Clipboard` and execute the method `PasteAction.OnUpdate()` is not STA despite the `main()` entry point is marked with `STAThreadAttribute`.

I've just created a PR to try to fix the issue (#259). 
I've tested the fix by using TabularEditor for a few hours and I did not experienced the issue any more.

**Comment by otykier** [comment]
Thanks a lot, @albertospelta! I have absolutely no experience with COM/interop/STA programming, so I would never have known where to look for the cause of this.

@kalmenk , I know you were facing a lot of these random crashes at times. Could you try the newest build to see if the problem is indeed solved? Thanks!
